exercise-28.js

```diff
@@ -0,0 +1,24 @@
+const prompt = require("prompt-sync")({ sigint: true });
+
+function countdown() {
+    let input = prompt('Enter a non-negative integer value (e.g. 0, 1, 2): ');
+
+    if (input < 0 || input % 1 != 0 || isNaN(input) === true) {
+        console.log("Exit Program");
+        return input;
+    } else {
+        console.log("Counting down");
+        let intervalId = setInterval(() => {
+            if (input < 0 || input % 1 != 0 || isNaN(input) === true) {
```

Comment:
I think it is safe to remove this, if the initial input is valid, then modifications here shouldn't make it invalid.